Selenium.Axe/Selenium.Axe/AxeResult.cs

```diff
@@ -24,15 +28,21 @@ public AxeResult(JObject result)
             JToken timestampToken = results.SelectToken("timestamp");
             JToken urlToken = results.SelectToken("url");
             JToken error = result.SelectToken("error");
+            JToken testEnvironment = results.SelectToken("testEnvironment");
+            JToken testEngine = results.SelectToken("testEngine");
+            JToken testEngineName = testEngine?.SelectToken("name");
+            JToken testEngineVersion = testEngine?.SelectToken("version");
 
             Violations = violationsToken?.ToObject<AxeResultItem[]>();
             Passes = passesToken?.ToObject<AxeResultItem[]>();
             Inapplicable = inapplicableToken?.ToObject<AxeResultItem[]>();
             Incomplete = incompleteToken?.ToObject<AxeResultItem[]>();
             Timestamp = timestampToken?.ToObject<DateTimeOffset>();
+            TestEnvironment = testEnvironment?.ToObject<AxeTestEnvironment>();
 
             Url = urlToken?.ToObject<string>();
             Error = error?.ToObject<string>();
+            TestEngine = $"{testEngineName?.ToObject<string>()} ({testEngineVersion?.ToObject<string>()})" ;
```

Comment:
The rest of this object format stays as close to the original axe format as possible; consider making a type with Name/Version properties here and leaving the string formatting to the report?

Reply:
Reasonable